src/Sarif.Converters/AndroidStudioConverter.cs

```diff
@@ -35,8 +35,8 @@ public AndroidStudioConverter()
         /// </summary>
         /// <param name="input">The Android Studio formatted log.</param>
         /// <param name="output">The SarifLog to write the output to.</param>
-        /// <param name="loggingOptions">Logging options that configure output.</param>
-        public override void Convert(Stream input, IResultLogWriter output, LoggingOptions loggingOptions)
+        /// <param name="dataToInsert">Logging options that configure output.</param>
```

Comment:
Update comment.

Reply:
Here and in similar places.

Reply:
Oops, good catch, thanks. Fixed everywhere.

---
In reply to: [201182951](https://github.com/Microsoft/sarif-sdk/pull/939#discussion_r201182951) [](ancestors = 201182951)